project_backend/Repositories/ItemRepository.cs

```diff
@@ -14,23 +15,52 @@ public ItemRepository(IDbConnection connection)
         }
         public IEnumerable<Item> GetItems()
         {
-            return _connection.Query<Item>("SELECT * FROM items");
+            return _connection.Query<Item>("SELECT * FROM items WHERE is_deleted = false");
```

Comment:
Neselektinam visko kaip *. Geriau surasyti, kokius stulpelius nori paimti.

To reikai tam, kad jei lentele baisiai didele, tai select * labai apkraus duomaze ir suletes darbas + gali istraukti jautrios informacijos, kurios geriau neistraukti (kaip passwordai ar pan)